frontend/src/components/FilterDrawerContent/FilterDrawerContent.tsx

```diff
@@ -0,0 +1,112 @@
+import { createStyles, Drawer, makeStyles, Theme, Typography } from '@material-ui/core'
+import React from 'react'
+import { spacing } from '../../styling'
+import { FilterSelector } from './FilterSelector'
+import { Dispatch } from '../../store'
+import { useDispatch } from 'react-redux'
+import { Title } from '../Title'
+import { CloseButton } from '../../buttons/CloseButton'
+
+const sortList = [
+  { value: 'name', filterName: 'Name' },
+  { value: 'state', filterName: 'State' },
+  { value: 'color', filterName: 'Color' },
+  { value: 'owner', filterName: 'Owner' },
+]
+const deviceList = [
+  { value: 'all', filterName: 'All' },
+  { value: 'active', filterName: 'Online' },
+  { value: 'inactive', filterName: 'Offline' },
+]
+const ownerList = [
+  { value: 'all', filterName: 'All' },
+  { value: 'me', filterName: 'Me' },
+  { value: 'others', filterName: 'Others' },
+]
+const sortFilter = [
+  { title: 'Sort', filters: sortList },
+  { title: 'Device State', filters: deviceList },
+  { title: 'Owner', filters: ownerList },
+]
+
+export function FilterDrawerContent({ open, close }: { open: boolean; close: (state: boolean) => void }): JSX.Element {
+  const { devices } = useDispatch<Dispatch>()
+  let owner = 'all',
+    state = 'all',
+    sort = 'name'
+  const css = useStyles()
+  const handleClose = () => {
+    close(false)
+  }
+  const handleClear = () => {
+    devices.set({
+      filter: state,
+      sort: sort,
+      owner: owner,
+      from: 0,
+    })
+    devices.fetch()
+  }
+
+  return (
+    <Drawer
```

Comment:
Let's change this to a <Collapse> and see if we can remove all the extra code trying to make the Drawer work inside a container.